src/GovUk.Education.ExploreEducationStatistics.Data.Api/appsettings.Development.json

```diff
@@ -13,6 +13,9 @@
       "Microsoft.EntityFrameworkCore.Database.Command": "Warning"
     }
   },
+  "FeatureManagement": {
+    "LocationHierarchies" : false
```

Comment:
I can't seem to see where this is causing features to be switched on and off?  Was expecting to see some mention of "LocationHierarchies" in the codebase other than in these files?

Reply:
Oh wait, sorry I see it!

Reply:
It's used in `SubjectMetaService.cs`. Check for `_featureManager.IsEnabledAsync`